.github/workflows/build_pip.yaml

```
@@ -0,0 +1,58 @@
+name: Editable build using pip and pre-release NumPy
+
+on: push
+
+permissions: read-all
+
+env:
+  PACKAGE_NAME: mkl_fft
+  MODULE_NAME: mkl_fft
+  TEST_ENV_NAME: test_mkl_fft
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+         shell: bash -el {0}
+
+    strategy:
+      matrix:
+        python: ['3.9', '3.10', '3.11', '3.12']
+        use_pre: ["", "--pre"]
+
+    steps:
+      - name: Install jq
+        shell: bash -l {0}
+        run: |
+          sudo apt-get install jq
+
+      - uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - uses: conda-incubator/setup-miniconda@v3
+        with:
+          use-mamba: true
+          mamba-version: "*"
+          channels: conda-forge
+          activate-environment: test
+          python-version: ${{ matrix.python }}
+
+      - name: Install MKL
+        run: |
+          conda install -c conda-forge mkl-devel mkl-service --override-channels
+          conda activate test
```

Are we going to use the "mamba" command here, since `use-mamba: true` was specified?
```suggestion
          mamba install mkl-devel mkl-service
```
I believe we don't need to activate the env and to pass the channels, because that was defined by default configuration in `conda-incubator/setup-miniconda` step.